addons/website_event/views/event_templates_page_misc.xml

```diff
@@ -68,4 +37,21 @@
     </t>
 </template>
 
+<template id="page_404" name="Page Not Found">
+    <t t-call="website_event.layout">
+        <div class="o_not_editable bg-100 pt40">
+            <div class="container">
+                <div class="alert alert-info text-center d-lg-flex justify-content-between align-items-center">
+                    <p class="m-lg-0 text-info">This event page does not exist, but you can create it as you are editor of this site.</p>
+                    <a role="button" class="btn btn-info" data-post_force-top-window="true" t-attf-href="/website/add/#{path}?redirect=1#{from_template and '&amp;template=%s' % from_template}">Create Page</a>
```

Comment:
Not sure about this feature, I will probably need to test a bit more to understand how that works because quickly trying this I end up with the incorrect url or can't create the page if the menu is already created (got a 405 Method Not Allowed)